app/Http/Controllers/User/ReportController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace App\Http\Controllers\User;
+
+use App\Models\Report;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\Auth;
+
+class ReportController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        //
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        //
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        //
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Request $request, $id)
+    {
+        $user = Auth::id();
+
+        $report = Report::where('user_id', $user)->where('post_id', $id)->first();
+
+        if (!$report) {
+            $report = Report::create([
+                'user_id'   =>  $request->get('user'),
+                'type'      =>  1,
+                'post_id' => $request->get('post_id'),
+            ]);
+        } else {
+            if ($request->get('type') == 1 ){
+                $report->update([
+                    'type' => 0,
+                ]);
+            } else {
+                $report->update([
+                    'type' => 1,
+                ]);
+            }
+        }
+
+        return response()->json([
+            'error' => false,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ReportController.php</br>**Line**: 99</br>Each line in an array declaration must end in a comma